### ISSUE TITLE
travis build failing due to qpdf

### ISSUE THREAD

**Issue by patrickmdnet** [jackwasey/icd#197]
I'm trying to get travis working on my pull #192.  When travis ran for me a month ago, I got away with this message:

https://travis-ci.org/github/jackwasey/icd/jobs/661837752
```
* checking compiled code ... OK
* checking sizes of PDF files under ‘inst/doc’ ... NOTE
Unable to find GhostScript executable to run checks on size reduction
```
That travis build passed.  Now, I get this message:
https://travis-ci.org/github/jackwasey/icd/jobs/667969517
```
* checking compiled code ... OK WARNING
‘qpdf’ is needed for checks on size reduction of PDFs
```

That WARNING triggers a check failure.
I looked at the most recent master branch travis check, and that failed in the same way, but it didn't fail on the WARNING. (It's not clear to me why it failed.)

https://travis-ci.org/github/jackwasey/icd/jobs/667028978
```
 * checking compiled code ... OK
 WARNING
‘qpdf’ is needed for checks on size reduction of PDFs
```

**Comment by patrickmdnet** [comment]
It turns out the error message is misleading.  If you pass `--as-cran` to `R check` and you have the envvar `_R_CHECK_DOC_SIZES_=FALSE`, that error will display regardless of whether qpdf can be found.  

I fixed this by setting `_R_CHECK_DOC_SIZES_=TRUE`.  I also had to set `_R_BUILD_COMPACT_VIGNETTES_=gs+qpdf` to avoid the fat vignettes warning.

Pull #198 has all changes.

**Comment by patrickmdnet** [comment]
Fixed with f4bca68c362d8acf6dc622d0f5403a0c67a8518a
